sncosmo/builtins.py

```diff
@@ -404,8 +410,24 @@ def tophat_bandpass_um(ctr, width, name=None):
                     ('4shooter2::i', 'bandpasses/4shooter2/I_4Shooter2.txt')]:
         _BANDPASSES.register_loader(name, load_bandpass_remote_aa,
                                     args=(fname,), meta=fourshooter_meta)
+
+wfirst_meta = {
+    'filterset': 'wfirst',
+    'retrieved': '13 Jun 2018',
+    'description': 'WFIRST filters from Hounsell et al. 2017 ' 
+    '(http://adsabs.harvard.edu/abs/2017arXiv170201747H)'}
+for name, fname in [('j129', 'wfirst/WFIRST_J129.dat'),
+                    ('w149', 'wfirst/WFIRST_W149.dat'),
+                    ('y106', 'wfirst/WFIRST_Y106.dat'),
+                    ('z087', 'wfirst/WFIRST_Z087.dat'),
+                    ('f184', 'wfirst/WFIRST_F184.dat'),
+                    ('h158', 'wfirst/WFIRST_H158.dat')]:
```

Comment:
The naming convention is updated, see the website PR, https://github.com/sncosmo/sncosmo.github.io/pull/23.